src/app/components/footer/footer.component.css

```
@@ -0,0 +1,6 @@
+footer{
+    flex-shrink: 0;
+    flex-grow: 0;
+    background: #eee;
+    padding: 3px 15px;
+  }
```

```suggestion
.footer{
    flex-shrink: 0;
    flex-grow: 0;
    background: #eee;
    padding: 3px 15px;
    position: fixed;
    bottom: 0;
  }
```

I Think this could be either a class or a Id field